plugins/woocommerce/src/Internal/Admin/Navigation/Telemetry.php

```
@@ -0,0 +1,103 @@
+		add_action( self::DAILY_BEACON_ACTION, array( $this, 'emit_daily_beacon' ) );
+
+		$this->maybe_schedule_daily_beacon();
+	}
+
+	/**
+	 * Fire toggled event with the new enabled state.
+	 *
+	 * @param string $old Old value.
+	 * @param string $new New value.
+	 */
+	public function on_flag_toggled( $old, $new ): void {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$old'. (undefined)

(UnusedFormalParameter)